Sources/Base.lproj/teaBASE.xib

```diff
@@ -1257,10 +1260,10 @@ Gw
         <window title="SSH Passphrase Window" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" releasedWhenClosed="NO" frameAutosaveName="" animationBehavior="default" id="Lio-kg-POV">
             <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES"/>
             <windowPositionMask key="initialPositionMask" leftStrut="YES" rightStrut="YES" topStrut="YES" bottomStrut="YES"/>
-            <rect key="contentRect" x="283" y="305" width="414" height="264"/>
-            <rect key="screenRect" x="0.0" y="0.0" width="1512" height="944"/>
-            <view key="contentView" id="Ye9-Xh-Sck">
-                <rect key="frame" x="0.0" y="0.0" width="413" height="306"/>
+            <rect key="contentRect" x="283" y="305" width="414" height="306"/>
+            <rect key="screenRect" x="0.0" y="0.0" width="1470" height="919"/>
+            <view key="contentView" misplaced="YES" id="Ye9-Xh-Sck">
```

Comment:
fricking `xib`s always doing this every other time you open them. Apple don’t give a shit. Nothing we can do.